docs/policies/vulnerability_management_policy.md

```diff
@@ -235,9 +239,9 @@ The vulnerability management team:
 
```

Comment:
Likely my fault for missing this - I think we ended up deciding on 'Revision history and decision records' as H2 here and then the rest comes underneath, could we add that in while we're here?